src/features/retrospective/ui/RetrospectiveCard.tsx

```
@@ -0,0 +1,63 @@
+        >
+          {totalCount}건
+        </span>
+      </div>
+      <span className='mt-1'>{description}</span>
+      <span className='text-[22px] text-[#222] font-semibold mt-3'>
+        {totalAmount.toLocaleString()}원
+      </span>
+      <ul className='flex flex-col gap-4 mt-8'>
+        {items.map((item) => (
+          <li className='flex flex-row'>
+            <CategoryTag tagName={item.category.name} size='small' />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)